plugins/modules/sns_topic_info.py

```diff
@@ -54,6 +54,11 @@
         type: complex
         returned: always
         contains:
+            content_based_deduplication:
+              description: Whether or not content_based_deduplication was set
+              returned: always
+              type: str
+              sample: "true"
```

Comment:
`version_added: 5.3.0` ?

Reply:
It's been there since the API started returning the value, it just wasn't documented